headjack-ui/src/pages/ChatPage.js

```diff
@@ -1,22 +1,50 @@
 import React, { useState, useCallback, useEffect } from "react";
 import useWebSocket, { ReadyState } from "react-use-websocket";
-import { TypeAnimation } from "react-type-animation";
 import { Player } from "@lottiefiles/react-lottie-player";
 import animatedYellowRobot from "../lottie/yellowRobot.json";
 import animatedPurpleRobot from "../lottie/purpleRobot.json";
 import Plot from "react-plotly.js";
 import DataTable from "../components/dataTable";
-
+const PeopleTable = ({ metadatas }) => {
+  return (
+    <table className="w-full border-collapse border border-gray-300">
+      <thead className="bg-gray-50">
+        <tr>
+          <th className="py-2 px-4 border border-gray-300">First Name</th>
+          <th className="py-2 px-4 border border-gray-300">Last Name</th>
+          <th className="py-2 px-4 border border-gray-300">Position</th>
+          <th className="py-2 px-4 border border-gray-300">Manager ID</th>
+          <th className="py-2 px-4 border border-gray-300">Hire Date</th>
+          <th className="py-2 px-4 border border-gray-300">Employee</th>
+          <th className="py-2 px-4 border border-gray-300">Description</th>
+        </tr>
+      </thead>
+      <tbody>
+        {metadatas.map((metadata, index) => (
+          <tr key={index} className="bg-white">
+            <td className="py-2 px-4 border border-gray-300">{metadata.first_name}</td>
```

Comment:
This design of consistently having a top level `utterance` and `metadata` attribute looks great! We can formalize these various metadata structures by creating a more rigid HSS spec.